src/server/scripts/poller.js

```diff
@@ -34,6 +34,11 @@ export const pollRepositories = (checker) => {
   let pollRepoLock = new AsyncLock();
   let locked_promises = [];
 
+  let poller_request_builds = false;
+  try {
+    poller_request_builds = JSON.parse(conf.get('POLLER_REQUEST_BUILDS'));
+  } catch (e) {}
```

Comment:
Linter is complaining about this empty catch block.

It would be good to at least log something here or add a comment why it's empty (comment should be enough for linter to pass).

Reply:
Sorry for letting it pass. It is fixed now, by logging an error message.